src/main/java/org/myrobotlab/swing/ServoGui.java

```diff
@@ -726,8 +726,8 @@ public void run() {
 
         mapInputSlider.setLowValue((int) servo.getMinInput());
         mapInputSlider.setHighValue((int) servo.getMaxInput());
-        mapOutputSlider.setLowValue((int) servo.getMinOutput());
-        mapOutputSlider.setHighValue((int) servo.getMaxOutput());
+        mapOutputSlider.setLowValue(servo.getMinOutput().intValue());

```

Comment:
this is odd syntax... semi-colons between args?

Reply:
?